src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,184 @@
+import { useEffect, useState, useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
 
 export const PeoplePage = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isDataFetched, setIsDataFetched] = useState(false);
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const sex = searchParams?.get('sex');
+  const centuries = searchParams?.getAll('centuries') || [];
+  const query = searchParams?.get('query') || '';
+  const centuriesArr: string[] = ['16', '17', '18', '19', '20'];
```

Comment:
we can move out of the component this array
and rename in uppercase as constant